pennathletics/models.py

```diff
@@ -13,4 +13,4 @@ def __init__(self, no, name, pos, ht, wt, yr, hometown):
         self.hometown = hometown
 
     def __repr__(self):
-        return "<Athlete #{}, {}>".format(self.no, self.name)
+        return ("<Athlete #{}, {},".format(self.no, self.name))
```

Comment:
ah - the original `__repr__` originally `print`ed instead of `return`ed, which was why there were parens added.